internal_test/mysql_test/mysql_test.go

```diff
@@ -206,24 +207,18 @@ func TestGetUserID(t *testing.T) {
 
 	t.Run(
 		"Valid_Users", func(t *testing.T) {
-			err := subject.AddUser("user_id_user", "user_id_user@example.com", "user_id_pass")
-			if err != nil {
+			if err := subject.AddUser("user_id_user", "user_id_user@example.com", "user_id_pass"); err != nil {
 				t.Fatal("Failed to add user -", err)
```

Comment:
Is this supposed to be an Error or a Fatal?

Reply:
Fatal is needed there since it would break the test, but below we want error bc its not "fatal"